package.json

```diff
@@ -82,52 +81,52 @@
 	},
 	"devDependencies": {
 		"@eslint-community/eslint-plugin-eslint-comments": "4.4.1",
-		"@eslint/js": "^9.15.0",
-		"@release-it/conventional-changelog": "^10.0.0",
+		"@eslint/js": "9.22.0",
+		"@release-it/conventional-changelog": "10.0.0",
 		"@types/dedent": "^0.7.2",
+		"@types/eslint-plugin-markdown": "2.0.2",
 		"@types/fs-extra": "^11.0.4",
-		"@types/node": "^22.10.1",
+		"@types/node": "22.13.10",
 		"@typescript-eslint/rule-tester": "^8.16.0",
 		"@typescript-eslint/utils": "^8.5.0",
-		"@vitest/coverage-v8": "^2.1.6",
-		"@vitest/eslint-plugin": "^1.1.12",
-		"console-fail-test": "^0.5.0",
-		"cspell": "^8.16.1",
+		"@vitest/coverage-v8": "3.0.9",
+		"@vitest/eslint-plugin": "1.1.38",
+		"console-fail-test": "0.5.0",
+		"create-typescript-app": "2.26.1",
+		"cspell": "8.17.5",
 		"dedent": "^1.5.3",
-		"eslint": "^9.15.0",
-		"eslint-doc-generator": "^2.0.0",
-		"eslint-plugin-eslint-plugin": "^6.3.2",
-		"eslint-plugin-jsdoc": "^50.6.0",
-		"eslint-plugin-jsonc": "^2.18.2",
-		"eslint-plugin-markdown": "^5.1.0",
-		"eslint-plugin-n": "^17.14.0",
-		"eslint-plugin-package-json": "^0.26.0",
-		"eslint-plugin-perfectionist": "^4.1.2",
-		"eslint-plugin-regexp": "^2.7.0",
-		"eslint-plugin-yml": "^1.15.0",
-		"husky": "^9.1.7",
-		"jsonc-eslint-parser": "^2.4.0",
-		"knip": "^5.38.2",
-		"lint-staged": "^15.2.10",
-		"markdownlint": "^0.37.0",
-		"markdownlint-cli": "^0.44.0",
-		"prettier": "^3.4.1",
-		"prettier-plugin-curly": "^0.3.1",
-		"prettier-plugin-packagejson": "^2.5.6",
-		"prettier-plugin-sh": "^0.15.0",
-		"release-it": "^18.0.0",
-		"sentences-per-line": "^0.3.0",
-		"should-semantic-release": "^0.3.0",
-		"tsup": "^8.3.5",
-		"typescript": "^5.7.2",
-		"typescript-eslint": "^8.16.0",
+		"eslint": "9.22.0",
+		"eslint-doc-generator": "2.1.0",
+		"eslint-plugin-eslint-plugin": "6.4.0",
+		"eslint-plugin-jsdoc": "50.6.8",
+		"eslint-plugin-jsonc": "2.20.0",
+		"eslint-plugin-markdown": "5.1.0",
+		"eslint-plugin-n": "17.16.2",
+		"eslint-plugin-package-json": "0.29.0",
+		"eslint-plugin-perfectionist": "4.10.1",
+		"eslint-plugin-regexp": "2.7.0",
+		"eslint-plugin-yml": "1.17.0",
+		"husky": "9.1.7",
+		"knip": "5.46.0",
+		"lint-staged": "15.5.0",
+		"markdownlint": "0.37.4",
+		"markdownlint-cli": "0.44.0",
+		"prettier": "3.5.3",
+		"prettier-plugin-curly": "0.3.1",
+		"prettier-plugin-packagejson": "2.5.10",
+		"prettier-plugin-sh": "0.15.0",
+		"release-it": "18.1.2",
+		"sentences-per-line": "0.3.0",
+		"tsup": "8.4.0",
+		"typescript": "5.8.2",
+		"typescript-eslint": "8.26.1",
 		"typescript54": "npm:typescript@5.4.5",
-		"vitest": "^2.1.6"
+		"vitest": "3.0.9"
 	},
 	"peerDependencies": {
-		"@typescript-eslint/parser": ">=6",
-		"eslint": ">=7",
-		"typescript": ">=4"
+		"@typescript-eslint/parser": ">=8",
+		"eslint": ">=9",
+		"typescript": ">=5"
```

Comment:
🐛 These should not be downgraded. There's no reason to restrict support in this PR. I'll fix this on the CTA side.